packages/system/src/Box.tsx

```diff
@@ -72,6 +72,19 @@ const isNotEmpty = (val: any) =>
 const ensureArray = <T extends any>(val?: T | T[]) =>
   Array.isArray(val) ? val : [val];
 
+/**
+ * Ensure that variant is an array and remove dot at the end of the string
+ * to set __default in theme without giving a default variant to the component
+ */
+const ensureCorrectVariant = (variant?: any | any[]) => {
```

Comment:
can only be string or string array, right?

Reply:
or undefined